### ISSUE TITLE
Single day becomes date range when using Moment Plugin

### ISSUE THREAD

**Issue by dcangulo** [fullcalendar/fullcalendar#4686]
It simply shows a single date in the header when using `timeGridDay` view.

But when I use the following custom view props it becomes date range instead:
```
views={{
  timeGrid: {
    titleFormat: "MMM Do YY"
  }
}}
```

**Default Output (without custom view props):** May 15, 2019
**Actual Output (with custom view props):** May 15th 19 – May 15th 19
**Expected Output:** May 15th 19

**Demo for reproduction:**
**Code:** https://codesandbox.io/s/kwjnpp5o6r
**Output:** https://kwjnpp5o6r.codesandbox.io/

**Comment by acerix** [comment]
Thanks for the detailed report, confirmed

**Comment by trfletch** [comment]
As a temporary workaround you can do the following:

```
resourceTimeGridDay: {
  titleFormat: 'dddd Do MMMM YYYY {}',
  titleRangeSeparator: ''
},
```

**Comment by arshaw** [comment]
This was fixed in 4.3.0 though please start using 4.3.1
Releases: https://github.com/fullcalendar/fullcalendar/releases

**Comment by ramavilli** [comment]
how to upgrade react version
